### ISSUE TITLE
Feature Request: Ability to Save Default Font Size for Text Tool

### ISSUE THREAD

**Issue by om-webninjaz** [flameshot-org/flameshot#3681]
### Feature Description

"Hello Flameshot Developers and Community,

I am proposing a feature to enhance the usability of Flameshot, specifically aimed at improving the text tool within the application. As a user who frequently utilizes the text tool for annotations, I find the current need to manually adjust the font size each time to be inefficient, especially when dealing with multiple screenshots in a session.

Feature Description:
The feature would allow users to set a default font size for the text tool that Flameshot would remember for all future uses. This setting could be adjusted in the configuration settings, enabling users to establish their preferred font size once and have it applied automatically each time they use the text tool.

Benefits:

Efficiency: Reduces the time spent on adjusting settings for each use, making the tool faster and easier to use.
User Experience: Enhances the overall user experience by allowing custom configurations that fit individual needs.
Consistency: Ensures consistent font sizing across different screenshots, which is particularly beneficial for users who create tutorials or documentation.
Potential Implementation:
A new option could be added to the Flameshot configuration UI under the 'Interface' or 'Annotations' section, where users can set their default font size. This setting would then be applied each time the text tool is selected.

Thank you for considering this feature. I believe it would make a valuable addition to Flameshot and greatly benefit users who rely on quick and consistent annotations.

Thank You.

**Comment by ClaudiaX68** [comment]
Yes please. At the moment it is very laborious to set font size, bold or the size of arrow line and such each time I open flameshot or even when adding a different layer and then text again it resets :(
add a new tab under User Interface -> Tools, where we can set
- default font, font-size, colour and style like bold for text
- default size for line, arrow, circle and such
- and let flameshot maybe remember during making a screenshot and adding different tools, the setting if we changed some

I really love flameshot and the ability now to use layers on a screenshot. Its the best tool, so thanks for all your improvemts so far :)
